src/lib/detectors.ts

```diff
@@ -439,4 +440,10 @@ function isSWAService(frameworks: Record<string, FrameworkMatch>, projectUrl: st
   }
   
   return false;
+}
+
+function isContainerAppService(frameworks: Record<string, FrameworkMatch>, projectUrl: string) {
+  console.log(`Url: ${projectUrl}`);
```

Comment:
remove?

Reply:
I copied that from isSWAService.. I guess we don't need it

Reply:
Done